media/themes/Lautus/lautus.msstyles/textfiles/NormalNormal.INI

```diff
@@ -1019,33 +1019,31 @@ BgType = ImageFile
 FillColorHint = 36 94 220
 ImageCount = 1
 ImageFile = Normal\TaskbarBackground.bmp
-SizingMargins = 1, 0, 23, 0
+SizingMargins = 1, 0, 6, 2
 SizingType = Stretch
-TRANSPARENT = true
-TRANSPARENTCOLOR = 255 0 255
```

Comment:
You confirm that these two transparency settings are not necessary for the `TaskBar.BackgroundButton`?

Reply:
Yes